user_docs/en/changes.t2t

```diff
@@ -42,9 +55,9 @@ What's New in NVDA
 - ``NVDA+Numpad Delete`` reports the location of the caret or focused object by default. (#13060)
 - ``NVDA+Shift+Numpad Delete`` reports the location of the review cursor. (#13060)
 - Added default bindings for toggling modifier keys to Freedom Scientific displays (#13152)
-- 'Baseline' is no longer reported via the report text formatting command (NVDA+F). (#11815)
+- "Baseline" is no longer reported via the report text formatting command (``NVDA+f``). (#11815)

```

Comment:
NVDA+F shortcut styling has been modified here:
- Code styling (fixed width) is used (adding double backquotes around the expression)
- The F has been switched from uppercase to lowercase.
Is it general styling recommendations to write documentation? Or is it just a matter of personal preference?

I have no strong reason to advocate for one styling or the other, but it's just to know if I need to change something in the future when writing documentation.

Thanks.

Reply:
>Code styling (fixed width) is used (adding double backquotes around the expression)
I think we are trying to do this consistently now.
If this formatting is missing from other entries for this release and future releases, we should try to update them.
I don't know if this convention has been documented anywhere - they are a bit of a developing standard.

>The F has been switched from uppercase to lowercase.

When documenting keyboard commands, "f" is the key name by convention, "F" tends to imply "shift+f".